requirements/frozen_docs.py3.txt

```diff
@@ -57,111 +61,112 @@ jupyter==1.0.0
 jupyter-client==5.2.3
 jupyter-console==5.2.0
 jupyter-core==4.4.0
-keystoneauth1==3.7.0
+keystoneauth1==3.9.0
 kwargify==2.2.0
 layered-yaml-attrdict-config==16.1.0
-lxml==4.2.1
+lxml==4.2.3
 manageiq-client==0.4.1
 MarkupSafe==1.0
 mccabe==0.6.1
 miq-version==0.1.4
 mistune==0.8.3
 mock==2.0.0
 monotonic==1.5
+more-itertools==4.2.0
 multi-key-dict==2.0.3
 multimethods.py==0.5.3
-navmazing==1.1.5
+navmazing==1.1.6
 nbconvert==5.3.1
 nbformat==4.4.0
 netaddr==0.7.19
 netifaces==0.10.7
 notebook==5.5.0
-ntlm-auth==1.1.0
+ntlm-auth==1.2.0
 oauthlib==2.1.0
 os-service-types==1.2.0
 oslo.i18n==3.20.0
-oslo.utils==3.36.2
+oslo.utils==3.36.3
 pandocfilters==1.4.2
 paramiko==2.4.1
 parsedatetime==2.4
-parso==0.2.1
-pbr==4.0.3
+parso==0.3.1
+pbr==4.1.0
 pdfminer.six==20170720
-pexpect==4.5.0
+pexpect==4.6.0
 pickleshare==0.7.4
-pika==0.11.2
-Pillow==5.1.0
+pika==0.12.0
+Pillow==5.2.0
 pluggy==0.6.0
 prettytable==0.7.2
-progress==1.3
+progress==1.4
 prompt-toolkit==1.0.15
-psycopg2==2.7.4
-ptyprocess==0.5.2
-py==1.5.3
-pyasn1==0.4.2
+psycopg2==2.7.5
+ptyprocess==0.6.0
+py==1.5.4
+pyasn1==0.4.3
 pycodestyle==2.3.1
 pycparser==2.18
 pycrypto==2.6.1
-pycryptodome==3.6.1
+pycryptodome==3.6.4
 pyflakes==1.6.0
 pygal==2.4.0
-PyGithub==1.39
+PyGithub==1.40
 Pygments==2.2.0
-PyJWT==1.6.3
+PyJWT==1.6.4
 PyNaCl==1.2.1
 pyOpenSSL==18.0.0
 pyparsing==2.2.0
-pytesseract==0.2.0
-pytest==3.4.1
+pytesseract==0.2.2
+pytest==3.6.0
 python-bugzilla==2.1.0
 python-dateutil==2.7.3
-python-glanceclient==2.11.0
-python-jenkins==1.0.0
-pytz==2018.4
+python-glanceclient==2.11.1
+python-jenkins==1.0.2
+pytz==2018.5
 pywinrm==0.3.0
-PyYAML==3.12
+PyYAML==3.13
 pyzmq==17.0.0
 qtconsole==4.3.1
 reg==0.11
 repoze.lru==0.7
-requests==2.18.4
+requests==2.19.1
 requests-ntlm==1.1.0
-requests-oauthlib==0.8.0
+requests-oauthlib==1.0.0
 requests-toolbelt==0.8.0
 riggerlib==3.1.5
 scp==0.11.0
 selenium==3.12.0
 selenium-smart-locator==0.2.0
 Send2Trash==1.5.0
-sentaku==0.6.2
+sentaku==0.6.4
 setuptools-scm==2.1.0
 shyaml==0.5.2
 simplegeneric==0.8.1
-simplejson==3.15.0
+simplejson==3.16.0
 six==1.11.0
 slumber==0.7.1
 smmap2==2.0.3
 snowballstemmer==1.2.1
 Sphinx==1.3.5
-sphinx-rtd-theme==0.3.1
-SQLAlchemy==1.2.7
+sphinx-rtd-theme==0.4.0
+SQLAlchemy==1.2.9
 stevedore==1.28.0
 tabulate==0.7.7
-taretto==0.1.1
+taretto==0.5
 terminado==0.8.1
 testpath==0.3.1
-tornado==5.0.2
+tornado==5.1
 traitlets==4.3.2
 tzlocal==1.5.1
-urllib3==1.22
+urllib3==1.23
 wait-for==1.0.9
 warlock==1.3.0
 wcwidth==0.1.7
 webencodings==0.5.1
 websocket-client==0.40.0
 Werkzeug==0.14.1
-widgetastic.core==0.21.3
-widgetastic.patternfly==0.0.34
+widgetastic.core==0.21.4
```

Comment:
@psav @RonnyPfannschmidt I don't think we want to bring this in now - it includes the `nav_click` FF60 workaround.